contribs/gmf/src/directives/partials/contextualdata.html

```diff
@@ -0,0 +1,2 @@
+This is the default template for contextual data.
+This should be overridden in applications.
```

Comment:
I just want to make sure that the integrators don't forget to override it.
I'm OK to remove it.